core/UrlHelper.php

```diff
@@ -155,6 +155,14 @@ public static function getArrayFromQueryString($urlQuery)
         if (strlen($urlQuery) == 0) {
             return array();
         }
+
```

Comment:
For example it is called quite often when detecting campaign parameters. We could reduce the number of calls to this method in some cases but even just executing it a second time causes overhead. In the end we want to avoid the calls to `Common::sanitizeInput` which is the actual slow part.